src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -511,6 +511,18 @@ class WP_HTML_Tag_Processor {
 	 */
 	protected $parser_state = self::STATE_READY;
 
+	/**
+	 * Indicates if the document is in quirks mode or no-quirks mode.
+	 *
+	 * @see self::QUIRKS_MODE
+	 * @see self::NO_QUIRKS_MODE
+	 *
+	 * @since 6.7.0
+	 *
+	 * @var string
+	 */
+	protected $compat_mode = self::NO_QUIRKS_MODE;
```

Comment:
Can we not bring over the previous comment here, and update it with the correction to the contradictory statement you mentioned?

Reply:
Is bb6d772 sufficient or do you think there should be more information on the property?

Reply:
having it on the constants is a good requirement. I can still see value in having at least a summary here, though I know we're not consistent everywhere in this. what I'm thinking about is someone hovering over the `$compat_mode` property and seeing documentation appear inline. for them, having a basic summary could help quickly inform them of whether or not they need to care about this. some IDEs do make it easy to follow the `@see` links, but not all do, and source code readers don't.

let me stew on it. maybe I can work at it some as well

Reply:
I've brought the comment over and updated the wording to make it clearer.

wanted to copy it here so that when people hover over `compat_mode` they get an instant clue on what it is and how it affects things